biomage/utils/constants.py

```diff
@@ -3,6 +3,8 @@
 STAGING = "staging"
 PRODUCTION = "production"
 
+DEFAULT_AWS_PROFILE = "default"
```

Comment:
So how do we map this default profile to the actual id of our account (242905224710), or do we need to  type it in every time?

Reply:
Do we get it from the credentials file locally?

Reply:
> The code looks good. Can you add a short section in the documentation, explaining how to run experiment download and the rds commands using a different account? Also add that by default, the commands will be run for the account that is under the `DEFAULT_AWS_PROFILE` profile name of the developer, stored in ~/.aws/profile.

The default values and where these profiles are stored is already explained if you do `--help` of any of these commands, so I'd rather not explain too much in the README. Still, I added an example of how to establish simultaneous rds connections.

Also I just pushed some changes to make it easier to establish simultaneous connections.